pom.xml

```diff
@@ -214,7 +214,6 @@
         <!-- Plugin configuration. There are many available options, see gwt-maven-plugin 
 					documentation at codehaus.org -->
         <configuration>
-          <workDir>${project.build.directory}/gwt/codeserver</workDir>
```

Comment:
sometimes I run into issues where this dir doesn't exist, and my plugin throws an error until I manually create it. I added this line in August, so I think I can safely remove it.